marshmallow/utils.py

```diff
@@ -335,15 +335,9 @@ def callable_or_raise(obj):
 
 
 def get_func_args(func):
```

Comment:
This can be simplified to:

```python
get_func_args = compat_get_func_args
"""Given a callable, return a list of argument names.
Handles `functools.partial` objects and callable objects.
"""
```